sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/ScalarFunctionImpl.java

```diff
@@ -109,6 +112,18 @@ public static Function create(Class<?> clazz, String methodName) {
    * @return created {@link Function} or null
    */
   public static Function create(Method method) {
+    return create(method, "");
+  }
+
+  /**
+   * Creates {@link org.apache.beam.vendor.calcite.v1_20_0.org.apache.calcite.schema.Function} from
```

Comment:
Noting that Calcite has a separate `Function` so they agree that defining your own is useful.